recipes/openms/meta.yaml

```diff
@@ -1,18 +1,19 @@
 {% set name = "OpenMS" %}
-{% set version = "2.1.0" %}
+{% set version = "2.2.0" %}
+{% set boost = "1.64" %}
 package:
   name: {{ name|lower  }}
   version: {{ version  }}
 
 source:
   fn: OpenMS-{{ version }}_src_contrib_doc.tar.gz
-  url: https://github.com/OpenMS/OpenMS/releases/download/Release{{ version }}/OpenMS-{{ version }}_src_doc_contrib.tar.gz
-  sha256: 36765c3a4c7a3277d71b9278505bd32f26c7030ffa6681a76344ca0024edf6a2
+  url: https://github.com/OpenMS/OpenMS/releases/download/Release2.2.0/OpenMS-{{ version }}-src.tar.gz
+  sha256: 61e7d24890abb6f462e2f026efb0dc88b84639b228d96e919c8074d3b11d286e
   
 build:
-    number: 1
-    string: boost1.62_{{PKG_BUILDNUM}}
-    skip: True  ## This package exceeds the travis build time, see the extra section.
+    number: 0
+    string: boost{{ boost }}_{{PKG_BUILDNUM}}
+    skip: False  ## This package exceeds the travis build time, see the extra section.
```

Comment:
@blankclemens if this still exceeds build time, we are allowed to upload it directly, in this case let me know and we do this on Monday.